path_test.go

```diff
@@ -95,6 +100,7 @@ var tests = []test{
 	{"//book[price='29.99']/title", "Harry Potter"},
 	{"//book/price[text()='29.99']", "29.99"},
 	{"//book/author[text()='Kurt Cagle']", "Kurt Cagle"},
+	{"//book/editor[text()]", []string{"Clarkson Potter", "\n\t\t"}},
```

Comment:
sorry I missed this earlier, but can you add unit test entries for the other editor tags you added?